### PR TITLE
chore: Add issue and PR templates

### PR DESCRIPTION
**Pull request by alimpens**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Describe the issue
+
+Thank you for contributing to the project! Please take some time to fill out this template, to help us quickly and smoothly handle your PR.
```
Comment (VincentSmedinga) [inline .github/issue_template.md:3]:
```suggestion
Thank you for contributing to the project!
Please use this template to help us handle your issue smoothly.
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Describe the pull request
+
+Thank you for contributing to the project! Please take some time to fill out this template, to help us quickly and smoothly handle your PR.
```
Comment (VincentSmedinga) [inline .github/pull_request_template.md:3]:
```suggestion
Thank you for contributing to the project!
Please use this template to help us handle your PR smoothly.
```

File: .github/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- Operating System: (e.g., Windows, macOS, Linux)
+- Version of the design system: (e.g., v1.2.3)
+
+## Checklist
+
+Before submitting your issue, please ensure you have done the following:
+
+- [ ] Searched the issue tracker to ensure the issue has not already been reported.
+- [ ] Verified that the issue can be consistently reproduced.
+- [ ] Checked if the issue is already documented.
```
Comment (VincentSmedinga) [inline .github/issue_template.md:29]:
```suggestion
- [ ] Search the issue tracker to ensure the issue has not already been reported.
- [ ] Verify that the issue can be consistently reproduced.
- [ ] Check if the issue is already documented.
```

File: .github/issue_template.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Describe the issue
+
+Thank you for contributing to the project! Please take some time to fill out this template, to help us quickly and smoothly handle your PR.
+
+## Summary
+
+(Briefly describe the issue encountered.)
+
+## Steps to Reproduce
```
Comment (VincentSmedinga) [inline .github/issue_template.md:9]:
```suggestion
## Steps to reproduce
```

File: .github/issue_template.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# Describe the issue
+
+Thank you for contributing to the project! Please take some time to fill out this template, to help us quickly and smoothly handle your PR.
+
+## Summary
+
+(Briefly describe the issue encountered.)
```
Comment (VincentSmedinga) [inline .github/issue_template.md:7]:
```suggestion
(Briefly describe the issue you encountered.)
```

File: .github/issue_template.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Step 1
+2. Step 2
+3. etc.
+
+## Environment
+
+- Browser: (e.g., Chrome, Firefox)
+- Operating System: (e.g., Windows, macOS, Linux)
+- Version of the design system: (e.g., v1.2.3)
```
Comment (VincentSmedinga) [inline .github/issue_template.md:21]:
```suggestion
- Browser: (e.g. Chrome, Firefox)
- Operating System: (e.g. Windows, macOS, Linux)
- Version of the design system: (e.g. v1.2.3)
```

File: .github/issue_template.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+## Checklist
+
+Before submitting your issue, please ensure you have done the following:
+
+- [ ] Searched the issue tracker to ensure the issue has not already been reported.
+- [ ] Verified that the issue can be consistently reproduced.
+- [ ] Checked if the issue is already documented.
+
+## Additional Notes
```
Comment (VincentSmedinga) [inline .github/issue_template.md:31]:
```suggestion
## Additional notes
```

File: .github/issue_template.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+- Mention if the issue is blocking development or critical for release.
```
Comment (VincentSmedinga) [inline .github/issue_template.md:33]: Will someone ever say it doesn’t? :)

File: .github/issue_template.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+- Provide links to any related issues or discussions.
+- If applicable, suggest a potential solution or workaround.
```
Comment (VincentSmedinga) [inline .github/issue_template.md:35]:
```suggestion
- Try suggesting a potential solution or workaround.
```

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+## What
+
+(What changes are being introduced in this PR?)
```
Comment (VincentSmedinga) [inline .github/pull_request_template.md:7]:
```suggestion
(What changes does this PR introduce?)
```

File: .github/pull_request_template.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+
+- [ ] Add or update unit tests
+- [ ] Add or update documentation
+- [ ] Add or update stories
+- [ ] Add or update exports in index.\* files
+- [ ] Start the PR title with a Conventional Commit prefix, [as explained here](https://github.com/Amsterdam/design-system/blob/main/documentation/publishing.md?plain=1#L11).
+
+## Additional notes
+
+- Please link to any related issues or discussions.
```
Comment (VincentSmedinga) [inline .github/pull_request_template.md:29]:
```suggestion
- Provide links to any related issues or discussions.
```

File: .github/pull_request_template.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+- Please link to the relevant story in the feature branch deploy.
```
Comment (VincentSmedinga) [inline .github/pull_request_template.md:30]:
```suggestion
- Add a link to the specific story in the feature branch deploy.
```

File: .github/issue_template.md (hunk 8 above)
Comment (alimpens) [reply .github/issue_template.md:34]: No, probably not. Shall we remove this point?

Comment (VincentSmedinga) [reply .github/issue_template.md:34]: Yes, let’s.

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**